docs/deployment/managed-scanning.md

```diff
@@ -11,53 +11,53 @@ tags:
 
 import InstallPrivateGitHubApp from "/src/components/procedure/_install-private-github-app.mdx"
 
-# Managed scanning
+# Semgrep Managed Scans
 
-**Semgrep managed scanning** enables you to add repositories to your Semgrep org in bulk without adding or changing your existing CI workflows. Similar to Semgrep CI workflows, managed scanning also integrates into developer workflows through PR comments.
+Add repositories to your Semgrep organization in bulk without adding or changing your existing CI workflows through **Managed Scans**. Similar to CI workflows, Managed Scans also integrates into developer workflows through PR comments.
 
 This is an alternative method to [adding Semgrep in CI](/deployment/add-semgrep-to-ci). Instead of adding a Semgrep job or workflow to your CI/CD pipeline, repositories are added to Semgrep AppSec Platform.
 
 ## Feature maturity and support
 
-- Managed scanning is in **public beta** for all existing Semgrep AppSec Platform users.
-- Managed scanning supports hosted GitHub (GitHub.com) and GitHub Enterprise Server plans.
+- Managed Scans is in **public beta** for all existing Semgrep AppSec Platform users.
+- It supports hosted GitHub (GitHub.com) and GitHub Enterprise Server plans.
     - This guide provides self-service enablement steps for **hosted GitHub plans**.
-    - To enable managed scanning on GitHub Enterprise Server, contact your technical account manager (TAM).
+    - To enable managed scans on GitHub Enterprise Server, contact your technical account manager (TAM).
 - Please leave feedback by either contacting your technical account manager (TAM) or through the **<i class="fa-solid fa-bullhorn"></i> Feedback** form in Semgrep AppSec Platform's navigation bar.
-- Managed scanning is available for all Semgrep products you have purchased, including:
+- Managed Scans is available for all Semgrep products you have purchased, including:
     - Semgrep Code
     - Semgrep Supply Chain
     - Semgrep Secrets
-- Managed scanning performs both full scans and diff-aware scans when a developer creates a pull request.
+- Semgrep performs both full and diff-aware managed scans when a developer creates a pull request.
```

Comment:
```suggestion
- Semgrep performs both full and diff-aware Managed Scans when a developer creates a pull request.
```
(though having this in title case feels weirder for some reason, idk)

Reply:
I think this is the case where Managed Scans != managed scans and it can stay lower case, since we're referring to the output of the feature, not the feature itself

Reply:
Agree @khorne3 and if it makes anybody feel better this is a common and funny problem in Tech - just today I used [GitHub Codespaces](https://docs.github.com/en/codespaces/overview) as an example (upper-case Codespaces != lower-case codespace).